### PR TITLE
docs(fix): revert prettier formatting on markdown files

### PR DESCRIPTION
**Pull request by tech4him1**
**Summary**

Prettier formatting our markdown files is causing bugs because of the differences between Gatsby's parser and Prettier's. Also, Prettier formats the inline code-blocks containing example CMS configs, but the formatting it uses doesn't really make much sense or match the suggested CMS config style.

It doesn't actually make much sense to format the docs anyway, since we use the CMS itself to edit/generate them.

**Test plan**

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit c0771ad1680edce98b24a5f2fd99613fc7265a38

https://deploy-preview-1612--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit c0771ad1680edce98b24a5f2fd99613fc7265a38

https://deploy-preview-1612--cms-demo.netlify.com

File: website/content/docs/widgets/list.md [hunk 1]
```diff
@@ -15,38 +15,41 @@ The list widget allows you to create a repeatable item in the UI which saves as
...
-    - label: 'Tags'
-      name: 'tags'
-      widget: 'list'
-      default: ['news']
+    - label: "Tags"
+      name: "tags"
+      widget: "list"
+      default: ["news"]
     ```
+
```
Comment (talves) [inline website/content/docs/widgets/list.md]: These blank line separators cause format errors. (needs review when build completes)

**Comment by tech4him1** [comment]
@talves Done. I already merged your #1606 changes in, thanks for the reminder.

**Pull request merged**